<ui version="4.0" >
 <class>QRXMLStruct</class>
 <widget class="QWidget" name="QRXMLStruct" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="Struct" >
      <column>
       <property name="text" >
        <string>Tag</string>
       </property>
      </column>
     </widget>
     <widget class="QTabWidget" name="TagInfos" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Attributes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QTreeWidget" name="TagAttributes" >
          <property name="rootIsDecorated" >
           <bool>false</bool>
          </property>
          <column>
           <property name="text" >
            <string>Attribute</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Content</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QTextEdit" name="TagContent" />
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
